ddl/index.go

```diff
@@ -484,6 +484,7 @@ type addIndexWorker struct {
 	priority  int
 
 	// The following attributes are used to reduce memory allocation.
+
```

Comment:
the empty line is generated by goword?

Reply:
I add it... if there isn't an empty line, goword will complain that this comment should begin with `defaultVals`

Reply:
how about following the suggestion of goword?

Reply:
What is the suggestion of goword?